Install-gui.ps1

```diff
@@ -0,0 +1,33 @@
+$ErrorActionPreference = "Stop"
+
+if ((Get-ChildItem env:VIRTUAL_ENV -ErrorAction SilentlyContinue) -eq $null)
+{
+    Write-Host "This requires the chia python virtual environment."
+    Write-Host "Execute '.\venv\Scripts\Activate.ps1' before running."
+    Exit 1
+}
+
+if ((Get-Command node -ErrorAction SilentlyContinue) -eq $null)
+{
+    Write-Host "Unable to find Node.js"
+    Exit 1
+}
+
+Write-Host "Running git submodule update --init --recursive."
+Write-Host ""
+git submodule update --init --recursive
+Write-Host "Running git submodule update."
+Write-Host ""
+git submodule update

```

Comment:
Shouldn't the first `update` call be sufficient?

Reply:
I'm not sure why install-gui.sh has this line. Shouldn't be necessary. I'll remove it from this script.